internal/feedback/feedback.go

```diff
@@ -73,7 +73,7 @@ func (cf ConstraintFeedback) LogFeedback(logger *log.Logger) {
 	if cf.Constraint != "" {
 		logger.Printf("  %v", GetUsingFeedback(cf.Constraint, cf.ConstraintType, cf.DependencyType, cf.ProjectPath))
 	}
-	if cf.LockedVersion != "" && cf.Revision != "" {
+	if cf.Revision != "" {
```

Comment:
Can you add a test for this change in [`TestFeedback_Constraint()`](https://github.com/golang/dep/blob/master/internal/feedback/feedback_test.go#L16) ?